test/test_helper.rb

```diff
@@ -9,7 +9,7 @@
 require "tempfile"
 require "debug"
 
-Minitest::Reporters.use!(Minitest::Reporters::SpecReporter.new(color: true))
+Minitest::Reporters.use!(Minitest::Reporters::DefaultReporter.new(color: true))
```

Comment:
Do we even need the minitest-reporters gem to use the default one? Or is it because minitest comes without colour by default?